nrf_security/Kconfig

```diff
@@ -1654,6 +1654,9 @@ config MBEDTLS_X509_CSR_WRITE_C
 
 endif # NRF_SECURITY_ADVANCED
 
+comment "PSA Native ITS"
```

Comment:
Yes it is. The comment separates PSA Native ITS section from others in Nordic Security menu.

Reply:
Also there will be a necessity of having PSA-specific configurations that are just using the actual crypto feature configurations from nrf_security

Right now the mbed TLS has a naming that keeps the defines MBEDTLS_XXXX_C regardless of existence of PSA. This may change in due time, but the menus selecting the specific features in a "human readable form" after PSA APIs are standard